subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/PluginUnderTestMetadata.java

```diff
@@ -55,7 +55,7 @@
     /**
      * The code under test. Defaults to {@code sourceSets.main.runtimeClasspath}.
      */
-    @Classpath
+    @Input
```

Comment:
This will produce a warning now.  Could you extract this as a getter and annotate it like you did with the script generator task?
https://github.com/gradle/gradle/blob/93a68297faf3681679e58253bd73e326f02fa9e4/subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/PluginUnderTestMetadata.java#L84-L84